guillotina/request.py

```diff
@@ -1,19 +1,136 @@
-import time
-import uuid
+from aiohttp.web_ws import WSMessage
```

Comment:
So we are still depending on aiohttp. Is it necessary? Could we remove the dependency?

Or was this decision done to make migration easier?

Any thoughts on migrating these into core guillotina?

Reply:
I should be easy to copy all the WS models to `guillotina/request.py`. I think that the only thing that is strongly coupled with aiohttp is the routing/urlmatchinfo logic

Reply:
yeah, i think this can be removed  quite easily